test/integration/puppetlabs/puppetserver/bootstrap_testutils.clj

```diff
@@ -39,6 +39,9 @@
 (def master-log-dir
   "./target/master-var/log")
 
+(def multithreaded
+  (or (= "true" (System/getenv "MULTITHREADED")) false))
```

Comment:
What's the purpose of wrapping this in an `or`?

Reply:
Good point, that's redundant.